### ISSUE TITLE
Use Implicit Type refactoring should detect if type is apparent in declaration for tuples

### ISSUE THREAD

**Issue by jcouv** [dotnet/roslyn#11094]
See [background](https://github.com/dotnet/roslyn/pull/10994#discussion_r62114452).

Arguably, the type is apparent in this case and so this test should pass. Currently the action is not available because `IsTypeApparentInDeclaration` doesn't think the type is apparent.

``` C#

        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseImplicitType), Trait(Traits.Feature, Test.Utilities.Traits.Features.Tuples)]
        public async Task SuggestVarOnLocalWithIntrinsicTypeTuple()
        {
            var before = @"class C { static void M() { [|(int a, string)|] s = (a: 1, ""hello""); } }";
            var after = @"class C { static void M() { var s = (a: 1, ""hello""); } }";

            await TestAsync(before, after, options: ImplicitTypeWhereApparent(), parseOptions: TestOptions.Regular.WithTuplesFeature(), withScriptOption: true);
        }
```

File `UseImplicitTypeTests.cs`

FYI for @balajikris

**Comment by balajikris** [comment]
sounds right to me. it looks like it may require some work to make the usevar codestyle work as expected in this case.

**Comment by jcouv** [comment]
Fix is merged for 15.6 preview 4
